src/styles/button.scss

```diff
@@ -38,6 +39,7 @@
 .button:hover,
 .button:focus {
   text-decoration: none;
+  width: 102%;
```

Comment:
in sass this can be written as 

```
.button { 
  &::focus,
  &::hover {
       ....css style
  }
}
```

Reply:
Also, the percentage should be between 0-100 - so this should be 100%

I'm not sure a width on pseudo-classes is necessary, but it's just a state for the element, not the actual element